tests/playwright/sanity/search-widgets.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from '@playwright/test';
+import { parallelTest as test } from '../parallelTest';
+import WpAdminPage from '../pages/wp-admin-page';
+
+test( 'Visible widgets should be shown in search result', async ( { page, apiRequests }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+	await wpAdmin.openNewPage();
+
+	// Act - search for a visible widget.
+	const widgetSearchBar = 'input#elementor-panel-elements-search-input';
+	await page.waitForSelector( widgetSearchBar );
+	await page.locator( widgetSearchBar ).fill( 'Spacer' );
+
+	// Assert - the widget should be shown in search result.
+	const widgetsInSearchResult = await page.$$( '#elementor-panel-elements .elementor-element-wrapper .elementor-element' );
```

Comment:
[page.$$ is discouraged.
](https://playwright.dev/docs/api/class-page#page-query-selector-all)
PLease use page.locator